config/build.gradle

```diff
@@ -19,15 +21,14 @@ dependencies {
 
     // --- Test Compile ---
     testImplementation project(path: ":core", configuration: "tests")
-    testImplementation group: "junit", name: "junit", version: junit_version
+    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter', version: junit_jupiter_version
```

Comment:
Why not
`testImplementation(group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: junit_jupiter_version)
testRuntimeOnly(group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: junit_jupiter_version)`

Reply:
Should only require the main junit package